federator/src/lib/utils.js

```diff
@@ -147,6 +148,13 @@ function checkIfItsInRSK(chainId = -1) {
         chainId === 33;
 }
 
+async function getHeartbeatPollingInterval(host) {
+    const web3 = new Web3(host);
+    const chainId = await web3.eth.net.getId();
+    const pollingInterval = [1, 30, 31, 42].includes(chainId) ? 1000 * 60 * 60 : null;
```

Comment:
only RSK should go here, as heartbeat won't be used on Ethereum to avoid the gas costs
Also instead of null return the config value as we do here https://github.com/rsksmart/tokenbridge/pull/200/files#diff-dd57c37b5d93d5439975278e78f26555e218dcb64b8f1d85e50e2ed32c8e3cd9R57 so all the poling logic is encapsulated
```suggestion
    const pollingInterval = [30, 31].includes(chainId) ? 1000 * 60 * 60 : config.runHeartbeatEvery * 1000 * 60;;
```